src/config.rs

```diff
@@ -39,6 +40,7 @@ impl Config {
             kafka_url,
             kafka_env,
             proto_registry_host,
+            value: yaml,
```

Comment:
You should only expose the value you need instead of the entire file.

Reply:
I made it private and only expose configs under connector id

Reply:
It should be removed

Reply:
Don't set the value here